VideoLocker/res/layout/row_course_list.xml

```diff
@@ -8,67 +8,186 @@
     <RelativeLayout
```

Comment:
I think we can remove the top-level `RelativeLayout`, as there is no need to nest multiple `RelativeLayout` containers.